app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -159,7 +172,12 @@ internal fun PackageInfo.isSystemApp(): Boolean {
 
 internal fun PackageInfo.allowsBackup(): Boolean {
     if (packageName == MAGIC_PACKAGE_MANAGER || applicationInfo == null) return false
-    return applicationInfo.flags and FLAG_ALLOW_BACKUP != 0
+
+    // At backup time, the system will filter out any apps that *it* does not want to be backed up.
+    // Now that we have switched to D2D, *we* generally want to back up as much as possible;
+    // part of the point of D2D is to ignore FLAG_ALLOW_BACKUP (allowsBackup). So, we return true.
+    // See Google Change ID 183147249.
```

Comment:
What does this name?

That looks like a bug number, or is that a gerrit change? Link would be better.

Reply:
It's referenced in Google's code. This is the only place I can find it other than that, though: https://developer.android.com/about/versions/12/reference/compat-framework-changes

Reply:
Ah then it's likely on their internal Gerrit

Reply:
Done